paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1701,4 +1744,12 @@ class PaparazziPluginTest {
   }
 
   private fun File.registerForDeletionOnExit() = apply { filesToDelete += this }
+
+  companion object {
+    private val CONFIG_ADAPTER = Moshi.Builder().build()!!.adapter(Config::class.java).indent("  ")
+    private val MATCHES_PATTERN = Correspondence.from<String, String>(
+      { actual, expected -> actual.matches(expected.toRegex()) },
+      "matches"
+    )
```

Comment:
TIL

Reply:
(I can't resolve it)

Reply:
wdym?

Reply:
There's no Resolve button for me, because I'm neither author, nor collaborator on this repo/PR. So I, and probably other contributors too, cannot resolve conversations.

![Screenshot_20240119-203708](https://github.com/cashapp/paparazzi/assets/2906988/f4ba95e9-5a1a-4323-88db-d24a2d071a69)

Reply:
ah understood.  no worries, it's not a requirement on this repo to resolve all discussions before merge.